/ros_controllers_node:
  ros__parameters:
    PID:
      linear_velocity:
        Kp: 0.5
        Ki: 0.02
        Kd: 0.0
        error_threshold: 0.1
        signal_limit: 0.5
      angular_velocity:
        Kp: 2.0
        Ki: 0.0
        Kd: 0.0
        error_threshold: 0.1
        signal_limit: 0.52359877559
    Stanley:
      V: 0.25  
      K: 0.8
      error_threshold: 0.05
      signal_limit: 0.52359877559
    PurePursuit:
      lookahead_distance: 0.30
      constant_velocity: 0.25
      error_threshold: 0.1
      signal_limit: 0.52359877559
    MPC:
      horizon: 2
      error_threshold: 0.10
      linear_velocity:
        signal_limit: 0.5
      angular_velocity:
        signal_limit: 1.52359877559
      Q: [10000000.0 , 10000000.0, 0.0] # State punishment matrix (x, y, theta)
      R: [10000.0, 0.0] # Input punishment matrix (v, w)
    vehicle_base_width: 0.14
    sleep_time: 100 # miliseconds
    csv_folder_name: "/home/furkan/controller_ws/src/ros_controllers/results/"